relayer/src/sql/create_tables.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS blocks (
+	block_height INTEGER NOT NULL,
+	block_hash TEXT NOT NULL,
+    block_header TEXT NOT NULL,
+	PRIMARY KEY (block_height, block_hash)
+);
+
+CREATE INDEX IF NOT EXISTS blocks_block_hash_idx ON blocks (block_hash);
+
+CREATE TABLE IF NOT EXISTS latest_block (
+    block_type INTEGER NOT NULL,
+	block_height INTEGER NOT NULL,
+    block_hash TEXT NOT NULL,
+    PRIMARY KEY (block_type)
```

Comment:
```suggestion
	block_type INTEGER NOT NULL,
	block_height INTEGER NOT NULL UNIQUE,
	block_hash TEXT NOT NULL UNIQUE,
	PRIMARY KEY (block_height)
```

Reply:
Nope there is a primary key is block height. As there would be only two records: Latest processed block, and latest finalized block

Reply:
So that's means that it's possible that we have equal value for both records in (block_height and block_hash) columns.